clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -853,6 +853,13 @@ def warn_cxxstruct_memaccess : Warning<
   "first argument in call to "
   "%0 is a pointer to non-trivially copyable type %1">,
   InGroup<NonTrivialMemcall>;
+
+def warn_cxxstruct_memaccess_relocatable
+    : Warning<"calling %0 with a pointer to a type %1 which is trivially "
+              "relocatable "
+              "but not trivially copyable; did you mean to call %2?">,
```

Comment:
```suggestion
              "but not trivially copyable; did you mean to call '%2'?">,
```
This one needs to be quoted manually because it’s just a `StringRef`.